R/selections.R

```diff
@@ -94,7 +94,8 @@
 NULL
 
 
-eval_select_recipes <- function(quos, data, info) {
+eval_select_recipes <- function(quos, data, info, allow_rename = FALSE) {
```

Comment:
I debated whether or not `allow_rename` should default to `TRUE` to be more consistent with tidyselect, but I think that it is better to default to `FALSE` as you have it here, since 99% of recipes steps will not allow renaming.